WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsJetpackSocialConnectionsContainer.kt

```diff
@@ -1,70 +1,47 @@
 package org.wordpress.android.ui.posts
 
 import android.content.res.Configuration
-import androidx.compose.foundation.layout.Column
-import androidx.compose.foundation.layout.PaddingValues
 import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.foundation.layout.padding
 import androidx.compose.material.Divider
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.tooling.preview.Preview
 import org.wordpress.android.models.PublicizeConnection
-import org.wordpress.android.models.PublicizeConnectionList
-import org.wordpress.android.ui.compose.components.buttons.PrimaryButton
 import org.wordpress.android.ui.compose.theme.AppThemeEditor
 import org.wordpress.android.ui.compose.unit.Margin
 import org.wordpress.android.ui.posts.social.PostSocialConnection
 import org.wordpress.android.ui.posts.social.compose.PostSocialMessageItem
-import org.wordpress.android.ui.posts.social.compose.PostSocialSharesText
 
 @Composable
-fun EditPostSettingsJetpackSocialContainer(
-    postSocialConnectionList: List<PostSocialConnection>,
-    showShareLimitUi: Boolean,
+fun EditPostSettingsJetpackSocialConnectionsContainer(
+    jetpackSocialConnectionDataList: List<JetpackSocialConnectionData>,
     shareMessage: String,
+    isShareMessageEnabled: Boolean,
     onShareMessageClick: () -> Unit,
-    remainingSharesMessage: String,
-    subscribeButtonLabel: String,
-    onSubscribeClick: () -> Unit,
-    modifier: Modifier = Modifier,
 ) {
-    Column(modifier = modifier) {
-        EditPostSettingsJetpackSocialConnectionsList(
-            postSocialConnectionList = postSocialConnectionList,
-        )
-        PostSocialMessageItem(
-            message = shareMessage,
-            modifier = Modifier
-                .padding(
-                    vertical = Margin.MediumLarge.value,
-                )
-                .fillMaxWidth(),
-            onClick = onShareMessageClick
-        )
-        if (showShareLimitUi) {
-            Divider()
-            PostSocialSharesText(
-                message = remainingSharesMessage,
+    EditPostSettingsJetpackSocialConnectionsList(
```

Comment:
🛑  It seems that the Column was removed here and this Composable is emitting multiple Composables without an actual Container now, which is unexpected, as the behavior will depend on the parent Composable, so it's not deterministic anymore.

Wrap the content of `EditPostSettingsJetpackSocialConnectionsContainer` in a `Column` or force this component to be only called inside `ColumnScope` by changing the signature to something like:
```kotlin
@Composable
fun ColumnScope.EditPostSettingsJetpackSocialConnectionsContainer(...) {
    // ...
}
```

Reply:
Thanks, @thomashorta ! It's fixed.